examples/with-tailwindcss/jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is unrelated to showing how to use tailwind so it shouldn't be in the example

Reply:
Thanks @timneutkens - I got rid of that commit and force pushed all of it back up.